src/source/source_simple_resonator.ts

```diff
@@ -0,0 +1,13 @@
+import { source_component_base } from "src/source/base/source_component_base"
+import { z } from "zod"
+
+export const source_simple_resonator = source_component_base.extend({
+  ftype: z.literal("resonator"),
+  frequency: z.union([z.string(), z.number()]),
+  load_capacitance: z.union([z.string(), z.number()]),
+  pin_count: z.number().default(3), // Set the default pin count to 3 for resonators
+  gender: z.enum(["male", "female"]).optional().default("male"),
```

Comment:
I don't think we will need gender for resonator.

cc: @seveibar

Reply:
nice catch @imrishabh18